core/src/types/operator/blocking_operator.rs

```diff
@@ -680,6 +680,98 @@ impl BlockingOperator {
         Ok(())
     }
 
+    ///
+    /// # Notes
+    ///
+    /// We don't support batch delete now.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use anyhow::Result;
+    /// # use futures::io;
+    /// # use opendal::BlockingOperator;
+    /// # fn test(op: BlockingOperator) -> Result<()> {
+    /// op.remove(vec!["abc".to_string(), "def".to_string()])?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn remove(&self, _paths: Vec<String>) -> Result<()> {
```

Comment:
We can call `delete` instead.

Reply:
we can call remove_via by iterators in `Vec` like this https://github.com/apache/incubator-opendal/blob/412cf2ba8cffeb10a09ff97439555cefb87f573c/core/src/types/operator/operator.rs#L1103

or we can use delete(_path) in the `remove` function. I am not sure which one

Reply:
remove_via -> remove -> delete